media-gfx/fontforge/fontforge-20220308.recipe

```diff
@@ -0,0 +1,139 @@
+CHECKSUM_SHA256="01e4017f7a0ccecf436c74b8e1f6b374fc04a5283c1d68967996782e15618e59"
+ADDITIONAL_FILES="fontforge.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2 !x86"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	fontforge$secondaryArchSuffix = $portVersion
+	app:fontforge
+	cmd:fontimage$commandSuffix
+	cmd:fontlint$commandSuffix
+	cmd:sfddiff$commandSuffix
+	lib:libfontforge$secondaryArchSuffix = 4 compat >= 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgdk_3$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpython3.10$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libspiro$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libcairo_gobject$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdk_3$secondaryArchSuffix
+	devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpython3.10$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libspiro$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
+	"
+
+BUILD()
+{
+	mkdir -p build && cd "$_"
+	cmake .. \
+		-DCMAKE_PREFIX=$appsDir/FontForge \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DENABLE_X11=OFF \
+		-DENABLE_PYTHON_SCRIPTING=OFF \
+		$cmakeDirArgs
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $developDir
+	cd build
+	make install
+
+	mkdir $appsDir
+	mv $binDir/fontforge $appsDir/FontForge
```

Comment:
This should probably be replaced with a symlink (or removed altogether) to retain the fontforge command.